frontend/src/components/common/form/StepForm/SteppedForm.tsx

```diff
@@ -17,27 +73,100 @@ export const SteppedForm = function<T extends object = {}>({
   validate,
   onSubmit,
   formikRef,
+  getTabs,
+  onAddTab,
+  onRemoveTab,
   ...rest
 }: ISteppedFormProps & FormikConfig<ISteppedFormValues<T>>) {
+  const [tabToDeleteId, setTabToDeleteId] = useState<number | undefined>();
   if (rest.persistable && !rest.persistProps) {
     throw new Error('SteppedForm: "persistProps" are required when "persistable" is true');
   }
 
+  if (!getTabs) {
+    initialValues.tabs = [{ activeStep: initialValues.activeStep }];
+  }
+
+  const tabTitle = (title: string, index: number) => {
+    return (
+      <>
+        <p>{title}</p>
+        <TooltipWrapper
+          toolTipId="remove-associated-parcel"
+          toolTip="Remove this associated parcel"
+        >
+          <FaWindowClose
+            size={15}
+            onClick={(e: any) => {
+              e.preventDefault();
+              e.stopPropagation();
+              setTabToDeleteId(index);
+            }}
+          ></FaWindowClose>
+        </TooltipWrapper>
+      </>
+    );
+  };
+  const getFormikTabs = getTabs ? getTabs : () => ['Tab 1'];
```

Comment:
@jdtoombs all stepped forms are now tabbed stepped forms. Forms with no getTabs prop will use one "hidden" tab. This allows me to reuse the stepped form without having to create a duplicate with step functionality.

Reply:
I've done some limited tests to confirm that this doesn't break the building form, but I'm hoping you can test this out as well.